src/test/regress/expected/multi_deparse_function.out

```diff
@@ -0,0 +1,311 @@
+(1 row)
+
+SELECT deparse_test($cmd$
+ALTER FUNCTION add PARALLEL UNSAFE
+$cmd$);
+                             deparse_test                             
+----------------------------------------------------------------------
+ ALTER FUNCTION function_tests.add(integer, integer) PARALLEL unsafe;
+(1 row)
+
+SELECT deparse_test($cmd$
+ALTER FUNCTION add PARALLEL RESTRICTED
+$cmd$);
+                               deparse_test                               
+--------------------------------------------------------------------------
+ ALTER FUNCTION function_tests.add(integer, integer) PARALLEL restricted;
+(1 row)
+
+SELECT deparse_test($cmd$
+ALTER FUNCTION add PARALLEL SAFE
+$cmd$);
+                            deparse_test                            
+--------------------------------------------------------------------
+ ALTER FUNCTION function_tests.add(integer, integer) PARALLEL safe;
+(1 row)
+
+-- The COST/ROWS arguments should always be numeric
+SELECT deparse_test($cmd$
+ALTER FUNCTION add COST 1234
+$cmd$);
+                             deparse_test                              
+-----------------------------------------------------------------------
+ ALTER FUNCTION function_tests.add(integer, integer) COST 1234.000000;
+(1 row)
+
+SELECT deparse_test($cmd$
+ALTER  FUNCTION add COST 1234.5
+$cmd$);
+                             deparse_test                              
+-----------------------------------------------------------------------
+ ALTER FUNCTION function_tests.add(integer, integer) COST 1234.500000;
+(1 row)
+
+SELECT deparse_test($cmd$
+ALTER FUNCTION add ROWS 10
+$cmd$);
+                             deparse_test                             
+----------------------------------------------------------------------
+ ALTER FUNCTION function_tests.add(integer, integer) ROWS  10.000000;
+(1 row)
+
+SELECT deparse_test($cmd$
+ALTER  FUNCTION add ROWS 10.8
+$cmd$);
+                             deparse_test                             
+----------------------------------------------------------------------
+ ALTER FUNCTION function_tests.add(integer, integer) ROWS  10.800000;
+(1 row)
+
+SELECT deparse_test($cmd$
+ALTER FUNCTION  add SET log_min_messages = ERROR
+$cmd$);
+                                   deparse_test                                    
+-----------------------------------------------------------------------------------
+ ALTER FUNCTION function_tests.add(integer, integer) SET log_min_messages = error;
+(1 row)
+
+SELECT deparse_test($cmd$
+ALTER FUNCTION  add SET log_min_messages TO DEFAULT
+$cmd$);
+                                     deparse_test                                     
+--------------------------------------------------------------------------------------
+ ALTER FUNCTION function_tests.add(integer, integer) SET log_min_messages TO DEFAULT;
+(1 row)
+
+SELECT deparse_test($cmd$
+ALTER FUNCTION  add SET log_min_messages FROM CURRENT
```

Comment:
what does from current entail, is this portable to remote workers that might have set the value to a different value?
From documentation:
```
SET FROM CURRENT saves the value of the parameter that is current when ALTER FUNCTION is executed as the value to be applied when the function is entered.
```

the qualification logic should resolve this and instead change the statement to a `ALTER FUNCTION ... SET ... TO current_value`.

Reply:
I solved this in deparsing logic for now. Need to move it into the associated `Qualify*()` funcs

Reply:
As discussed; please revert to deparse `SET FROM CURRENT` and open an issue to solve this later.
In #3041 I raise an error when this syntax is applied on a distributed function with some information on how the user can work around this limitation.

Reply:
I reverted my changes and I will deparse the `SET FROM CURRENT` and I will open an issue and link it to here